package.json

```diff
@@ -49,11 +50,13 @@
     "body-parser": "^1.19.0",
     "commitizen": "^4.2.4",
     "cors": "^2.8.5",
+    "cross-env": "^7.0.3",
     "cz-conventional-changelog": "3.3.0",
     "express": "^4.17.3",
     "express-rate-limit": "^6.3.0",
     "follow-redirects": "^1.15.1",
     "got": "^11.8.3",
+    "headers-polyfill": "^3.1.0",
```

Comment:
Moving it to dev dependencies gets it bundled, is that right? 

I wonder if we have any issues with `headers-polyfill` in this regard. I thought of bundling ESM-only dependencies as a way to enable this library for legacy consumers. But headers polyfill distributes like both CJS and ESM, does it not? 🤔

Reply:
Indeed it does distribute both CJS and ESM

Reply:
Let's move it back to `dependencies` then?